docs/source/release/v6.10.0/Framework/Algorithms/New_features/35710.rst

```diff
@@ -0,0 +1 @@
+- Added a new peak finding strategy named AllPeaksNSigma to FindSXPeaks :ref:`FindSXPeaks <algm-FindSXPeaks-v1>` algorithm.
```

Comment:
Sorry not to mention it before, but I think we should credit the author of SXD2001 for the idea of using NSigma as a threshold (albeit in SXD2001 the peak finding is done in 3D). 
`Gutmann, M. J. (2005). SXD2001. ISIS Facility, Rutherford Appleton Laboratory, Oxfordshire, England.`

Reply:
Perhaps you could add this reference in a comment in `NSigmaPeaksStrategy::getAllNSigmaPeaks`?